### PR TITLE
Adds initial documentation on OGNL cache configuration

### PR DESCRIPTION
**Pull request by JCgH4164838Gh792C124B5**
Hello Struts Development Team.

This is an initial attempt to add some documentation on OGNL cache configuration.  Not sure if the templating structure is OK or not.  Please advise if it can be modified and of use for the documentation site.

Regards.

### CODE REVIEW

File: source/core-developers/ognl-cache-configuration.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+```
+or
+```
+<constant name="struts.ognl.enableExpressionCache" value="false"/>
+```
+
+Disabling the OGNL expression cache can have a negative impact on performance, but 
+disabling the cache can limit memory overhead for the application (a trade-off).
+
+As of Struts 2.6, additional configuration properties have been made available to control the
```
Comment (lukaszlenart) [inline source/core-developers/ognl-cache-configuration.md:33]: Could you use "6.0.0" term?

File: source/core-developers/ognl-cache-configuration.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+```
+<!-- Set expression and BeanInfo caches to size 1000 with LRU cache mode -->
+<constant name="struts.ognl.enableExpressionCache" value="true"/>
+<constant name="struts.ognl.expressionCacheMaxSize" value="1000"/>
+<constant name="struts.ognl.expressionCacheLRUMode" value="true"/>
+<constant name="struts.ognl.beanInfoCacheMaxSize" value="1000"/>
+<constant name="struts.ognl.beanInfoCacheLRUMode" value="true"/>
+```
+
+As of Struts 2.6, developers may choose to implement their own OGNL cache factories,
```
Comment (lukaszlenart) [inline source/core-developers/ognl-cache-configuration.md:76]: Same here, please use "6.0.0" to avoid confusing users

**Review by lukaszlenart — CHANGES_REQUESTED** [review]
Looks good, just two minor things and I'm good :)

**Comment by JCgH4164838Gh792C124B5** [comment]
Hi.  The two requested changes have been made.  Please let me know if any other changes are needed.

**Review by lukaszlenart — APPROVED** [review]

**Comment by lukaszlenart** [comment]
Thanks, LGTM 👍

**Pull request merged**